src/TorSharp/Tools/SimpleToolRunner.cs

```diff
@@ -34,26 +34,18 @@ public Task StartAsync(Tool tool)
                 startInfo.EnvironmentVariables[pair.Key] = pair.Value;
             }
 
-            Process process = Process.Start(startInfo);
+            Process process = Process.Start(startInfo) ?? 
```

Comment:
Nice catch. I didn't know `Process.Start` could return null but I see that in the docs now.